ninja/pagination.py

```diff
@@ -295,15 +297,19 @@ def decode_cursor(cls, encoded_cursor: str | None) -> Cursor:
 
                 position = tokens.get("p", [None])[0]
             except (TypeError, ValueError):
-                raise ValueError(_("Invalid cursor."))
+                raise ValueError(_("Invalid cursor.")) from None
```

Comment:
is it necessary to raise from None?

Reply:
shouldn't we raise from TypeError or ValueError?

Reply:
```python
 except (TypeError, ValueError) as e:
                raise ValueError(_("Invalid cursor.")) from e
```
                
```

Reply:
I suppose `__cause__` is semantically correct, though the underlying exception will be in `__context__` regardless. Let me know which is preferred.